views/sidebars/carousel.handlebars

```diff
@@ -1,19 +1,52 @@
 <a href="#" class="backButton button btn-secondary"><i class="fa fa-back"></i> Back</a>
 
-<h5>Image 1</h5>
-
-<form method="PUT" action="/api/customization/carousel" class="customizationForm">
+<form  method="PUT" action="/api/customization/carousel/{{index}}" class="customizationForm {{data.option}}">
```

Comment:
The customization for carousel is not an array, you can remove {{index}}

Reply:
I removed index.  Having trouble with the original input code I overrode with the body code